packages/stacked/example/lib/app/app.dart

```diff
@@ -19,7 +19,7 @@ import 'package:stacked_themes/stacked_themes.dart';
     MaterialRoute(page: HomeView, initial: true),
     MaterialRoute(page: BottomNavExample),
     MaterialRoute(page: StreamCounterView),
-    CupertinoRoute<Map<String, List<String>>>(page: DetailsView),
+    CupertinoRoute<Map<String, List<String>>?>(page: DetailsView),
```

Comment:
To activate the type inference we have to add it here right?

If the above is right what happen when we change one of the class parameters?